ambient_toolbox/management/commands/create_translation_file.py

```diff
@@ -43,7 +43,7 @@ def handle(self, *args, **options):
 
         call_command("makemessages", "-l", language, "--no-location", "--no-wrap")
```

Comment:
I actually like the "location" as helper for the translators. Why do you remove it by default? Can I override it if I want to?

Reply:
Try working with it, then you'll stop liking it. It will cause merge conflicts ALL the time because you'll often change code which then leads to a new line in the location. Was one of the main drivers we built this thing.